ECDSA-adaptor.md

```diff
@@ -0,0 +1,209 @@
+# ECDSA adaptor signatures
+
+DLCs work by conditioning a transaction on the revelation of an oracle's signature .
+Concretely the approach is to encrypt a transaction signature such that the oracle's released signature is the decryption key.
+Once the oracle has revealed a signature, either party may use it to decrypt the corresponding transaction signature and broadcast the transaction.
+By this method an oracle's view of a real world event enforces a particular distribution of funds between two parties according to their private bet.
+
+## Adaptor Signatures
+
+*Adaptor signatures* are an efficient form of verifiable signature encryption that can achieve our required structure.
+Given a anticipated signature `Y` we may encrypt a signature (for the purposes of this document, an ECDSA signature) under `Y` such that once a party learns the discrete logarithm of `Y` they can decrypt it and get a valid ECDSA signature.
```

Comment:
nit: `Y` isn't a signature per se, bur rather a commitment to a signature, or a public key derived from an anticipated signature

Reply:
I think `Y` should just be "public key"? Just a typo I think.

Reply:
The name I use for the commitment/public key is "anticipated signature". It came from  Tadge: https://diyhpl.us/wiki/transcripts/discreet-log-contracts/.

I *think* I am using it correctly here.

Reply:
From that transcript, it looks like it's ambiguous as to whether "Anticipated Signature" refers to `Y` or just the scheme in general. However, my point might be a little pedantic and I don't think it matters all that much, and I can see the utility of having a shorthand like "Anticipated Signature."